pkg/crc/machine/bundle/repository.go

```diff
@@ -38,7 +38,7 @@ func (repo *Repository) Get(bundleName string) (*CrcBundleInfo, error) {
 		return nil, errors.Wrapf(err, "could not find cached bundle info in %s", path)
 	}
 	jsonFilepath := filepath.Join(path, metadataFilename)
-	content, err := os.ReadFile(jsonFilepath)
+	content, err := os.ReadFile(filepath.Clean(jsonFilepath))
```

Comment:
`filepath.Join` already calls `filepath.Clean` https://pkg.go.dev/path/filepath#Join so it's dubious that this fixes anything. It's also not clear exactly what exact scenario we want to protect against.